runtime/testing/browser/test-utility.ts

```diff
@@ -0,0 +1,23 @@
+export interface SkyAppTestUtilityEventArgs {
```

Comment:
I think I'd call this `SkyAppTestUtilityEventOptions` instead.  Ending a class name with `EventArgs` is a common pattern for classes used to pass arguments into an event handler, which is not the case here.  I'd also move it to a separate file so we're only exporting one type from this file.